### ISSUE TITLE
fetch/possible-values: how to supply path?

### ISSUE THREAD

**Issue by jmacneal** [intermine/imcljs#28]
I'd like to be able to use imcljs.fetch/possible-values to explore the database and auto-generate a parser grammar. Digging into the source, I can't really reason about the format expected for the _path_ argument ("organism.name"?). Everything I've tried results in a 400 error ("error": "Missing parameter: 'path'). I haven't quite been able to diagnose the exact issue (in io/get-body-wrapper- I believe), but I have re-implemented the function myself using clj-http.

**Comment by yochannah** [comment]
Hey @jmacneal! Sorry, I missed this notification until now. Please feel free to ping me if I'm that slow again :confused: - thanks for making this issue! 

Would you be able to PR or share your re-implementation by any chance?

**Comment by jmacneal** [comment]
Yeah, I've been meaning to. Just been extremely busy getting up to speed
with my new job.
